docs/CHANGELOG.asciidoc

```diff
@@ -28,6 +28,360 @@ This section summarizes the changes in each release.
 
 coming[7.2.0]
 
+
+[float]
+[[breaking-7.2.0]]
+=== Breaking Changes
+
+See <<breaking-changes-7.2, Breaking changes in 7.2>> for more information.
+
+Index Patterns::
+* Removes support for time-based interval index patterns with migration {pull}35262[#35262]
+
+[float]
+[[enhancement-7.2.0]]
+=== Enhancements
+Canvas::
+* Adds custom elements {pull}34140[#34140]
+* Adds static tags for elements {pull}28779[#28779]
+* Adds autoplay pages in fullscreen {pull}35981[#35981]
+* Adds new workpad templates {pull}35745[#35745]
+* Adds Ctrl + Enter shortcut for evaluating expressions {pull}36138[#36138]
+* Allows grouping of filters and group selection in the `filters` function {pull}34889[#34889]
+
+Design::
+* Upgrades EUI to 9.2.1 and improves dark mode {pull}32728[#32728]
+
+Elasticsearch UI::
```

Comment:
Elasticsearch UI is a team. The app is Management, so this should go into management section